### PR TITLE
Allow CC for ad-hoc compliance emails <PD-1603>

### PR DESCRIPTION
**Pull request by tdphillips**
Tests passed locally.

Test coverage is spotty, but I'm slowly increasing it.

### CODE REVIEW

File: redirect/helpers.py [hunk 1]
```diff
@@ -607,6 +612,9 @@ def send_response_to_sender(new_to, old_to, email_type, guid='', job=None,
         old_to = old_to[0]
     email = EmailMessage(from_email=settings.DEFAULT_FROM_EMAIL,
                          to=new_to)
+    if cc:
```
Comment (aspidites) [inline redirect/helpers.py]: What happens if you set email's cc to None? If its the same effect as not specifying it at all, then this if clause will be redundant.

File: redirect/tests/test_views.py [hunk 2]
```diff
@@ -1064,6 +1064,10 @@ def test_bad_email(self):
         self.assertEqual(len(mail.outbox), 0)
 
     def test_bad_guid_email(self):
+        """
```
Comment (aspidites) [inline redirect/tests/test_views.py]: Yay, doc strings!

File: redirect/helpers.py (hunk 1 above)
Comment (tdphillips) [inline redirect/helpers.py]: I'll check as soon as my tests finish.

File: redirect/tests/test_views.py (hunk 2 above)
Comment (tdphillips) [inline redirect/tests/test_views.py]: This is but a fraction of the undocumented tests, however. :sob:

File: redirect/helpers.py (hunk 1 above)
Comment (tdphillips) [inline redirect/helpers.py]: EmailMessage.cc is defined as a list or tuple.

Comment (aspidites) [inline redirect/helpers.py]: Ah. then `cc = cc or []`. Then the if is redundant.

Comment (aspidites) [inline redirect/helpers.py]: Actually, tuples are immutable, so why not change the default argument to ()? Then you could remove the if statement.

Comment (tdphillips) [inline redirect/helpers.py]: Done.

**Comment by aspidites** [comment]
Why is this marked to not merge?

**Comment by tdphillips** [comment]
Waiting on something brought up on Jira.

File: redirect/helpers.py (hunk 1 above)
Comment (tdphillips) [inline redirect/helpers.py]: This whole conversation proved moot. We're no longer CCing people on this particular email.

**Comment by aspidites** [comment]
Code looks good. Easily testable locally, or are we better to try this out in QC, @tdphillips?

File: redirect/helpers.py (hunk 1 above)
Comment (aspidites) [inline redirect/helpers.py]: Oh joy. I was wondering if I just glared over the cc code this morning or not. That makes sense now.

**Comment by tdphillips** [comment]
Neither, unless we're going to set up MX records for QC.

**Pull request merged**